docs/develop/_changelog.md

```diff
@@ -1,4 +1,23 @@
+# Changelog
+
+```{include} ../../CHANGELOG.md
+:end-before: <!-- towncrier release notes start -->
```

Comment:
This allows pasting the changelog draft between the top note and the released changelog.